witness/src/Dscp/Snowdrop/Expanders.hs

```diff
@@ -100,28 +112,42 @@ seqExpandersPublicationTx =
                 }
             } -> do
                 maybePub <- queryOne (PublicationsOf ptAuthor)
-
-                let change = if isJust maybePub then Upd else New
-
-                return $ mkDiffCS $ Map.fromList
-                  [ PublicationsOf  ptAuthor ==> change (LastPublication ptBlock)
-                  , PublicationHead ptBlock  ==> New    (PublicationNext ptPrevBlock)
-                  ]
+                queryOne (AccountId ptAuthor) >>= \case
+                    Just (account@ Account {..}) -> do
+                        let change = if isJust maybePub then Upd else New
+                        return $ mkDiffCS $ Map.fromList
+                          [ PublicationsOf  ptAuthor ==> change (LastPublication ptBlock)
+                          , PublicationHead ptBlock  ==> New    (PublicationNext ptPrevBlock)
+                          , AccountId ptAuthor ==>
```

Comment:
Wait, but who gets the fee?

Reply:
Same question. Nobody gets fee in money transfer as well.

Reply:
How do we know current block leader? Should I take it from slot info?

Reply:
So, question "how" remains.

Reply:
As an argument.

Reply:
We can simply burn the fee for alpha release, who even cares, Cardano mainnet is still doing this.

Reply:
Too late, send it to issure in the last commit.